src/test/java/org/springframework/integration/dsl/test/file/FileTests.java

```diff
@@ -307,7 +309,11 @@ public IntegrationFlow tailFlow() {
 		@Bean
 		public IntegrationFlow fileReadingFlow() {
 			return IntegrationFlows
-					.from(s -> s.file(tmpDir.getRoot()).patternFilter("*.sitest"),
+					.from(s -> s.file(tmpDir.getRoot())
+									.patternFilter("*.sitest")
+									.useWatchService(true)
```

Comment:
Interesting: http://stackoverflow.com/questions/9588737/is-java-7-watchservice-slow-for-anyone-else

`SensitivityWatchEventModifier.HIGH`